docs/source/pythonapi/examples/tally-arithmetic.ipynb

```diff
@@ -486,9 +482,9 @@
    "source": [
     "# Thermal Flux Utilization tallies\n",
     "fuel_therm_abs_rate = openmc.Tally(name='fuel therm. abs. rate')\n",
-    "fuel_therm_abs_rate.add_score('absorption')\n",
-    "fuel_therm_abs_rate.add_filter(openmc.Filter(type='energy', bins=[0., 0.625e-6]))\n",
-    "fuel_therm_abs_rate.add_filter(openmc.Filter(type='cell', bins=[fuel_cell.id]))\n",
+    "fuel_therm_abs_rate.scores = ['absorption']\n",
+    "fuel_therm_abs_rate.filters = [openmc.Filter(type='energy', bins=[0., 0.625e-6]),\n",
```

Comment:
Up to you whether to use `append` here, my OCD self is less opinionated if the continuation line length is of the same order as the preceding line :-)